WeCantSpell.Hunspell/WeCantSpell.Hunspell.csproj

```diff
@@ -0,0 +1,69 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard1.3;net451</TargetFrameworks>
```

Comment:
This might change, but for the next release I think this is what I am going with. The old versions were netstandard2.0;netstandard1.3;net45 which has a slight difference in version of ... "net001" :trollface: . Technically, this means I think I need to increment the version to a new major version, but I'm a rebel so I may just do this as a minor increment. The reason for this is that I can't get net45 to work on my system 🤷‍♂️.